tools/az-auth.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+
+az login --allow-no-subscriptions
```

Comment:
Maybe, best approach for this it will something like:

```suggestion
az login --allow-no-subscriptions --tenant-id $tenantId
```

To enforce tenantId we will to use; otherwise it will use the one defined as primary for your account -which it hasn't to be the one you want to use- and you will have to run same command again but with the right tenant